src/main/java/Duke.java

```diff
@@ -1,10 +1,13 @@
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Scanner;
```

Comment:
Great job importing only the libraries you need!

Reply:
Thanks! :)